README.md

```diff
@@ -51,31 +51,31 @@ Requirements: MonoGame + Visual Studio
 
 1. Download and install **Visual Studio 2017** (2015 is not supported) and **NET Framework 4.6**
 
-2. Clone whole repository:
+2. Clone the whole repository:
```

Comment:
Actually, "Clone the repository:" would be better

Reply:
True.